### DISCUSSION TITLE
NuGet upload

### DISCUSSION THREAD

**Discussion by unconverged** [Giorgi/DuckDB.NET#51] [General]
Maybe it's time to upload a new version to NuGet? We now have Db parameters which is a major improvement worth a new version. Or at least decide which issues need to fixed before a release.

**Comment by Giorgi** [comment]
Hello @IvaYan I think it would be great if prepared statements supported other primitive types too such as date and time.

**Reply by unconverged** [reply]
Indeed, it would be great. 

I think, it would also be great if you created a set of issues about what is missing and need to be implemented for the next version. This would let us monitor the progress. It may also be a good idea to specify (with labels) which issues are required for the next version and which are recommended, but not required. Such classification is needed because if everything is planned for the next version, then it will never be released.
